packages/js/src/generators/release-version/utils/resolve-version-spec.spec.ts

```diff
@@ -1,5 +1,5 @@
-import { join } from 'path';
 import { resolveVersionSpec } from './resolve-version-spec';
+import { joinPathFragments } from 'nx/src/utils/path';
```

Comment:
```suggestion
```

Reply:
I don't understand this comment
do you want me to delete `joinPathFragments` imports?